pallets/settlement/src/lib.rs

```diff
@@ -412,7 +412,7 @@ decl_error! {
 decl_storage! {
     trait Store for Module<T: Trait> as StoCapped {
         /// Info about a venue. venue_id -> venue_details
-        VenueInfo get(fn venue_info): map hasher(twox_64_concat) u64 => Venue;
+        pub VenueInfo get(fn venue_info): map hasher(twox_64_concat) u64 => Venue;
```

Comment:
```suggestion
        pub VenueInfo get(fn venue_info): map hasher(twox_64_concat) u64 => Option<Venue>;
```
Will generally make the other code nicer.

Reply:
@maxsam4 thoughts?

Reply:
Using `Option` sounds like a good idea. Substrate strips the `option` anyway so using option here is basically a way to get better API.